proto/query.proto

```diff
@@ -5,6 +5,8 @@ syntax = "proto3";
 
 package query;
 
+option java_package="com.youtube.vitess.grpc.proto";
```

Comment:
I would put the protos in a non-grp package, like:
com.youtube.vitess.proto
and the service ones for grpc in:
com.youtube.vitess.proto.grpc